dotnet/samples/ApplicationInsightsExample/ApplicationInsightsExample.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" />
+    <PackageReference Include="Azure.Monitor.OpenTelemetry.Exporter" />
```

Comment:
The comment is related to the project and folder name rather than to the current line. I would suggest renaming the project and its corresponding folder to something more abstract than AppInsight or OpenTelemetry, such as TelemetryExample.